dix/req.c

```diff
@@ -1799,6 +1799,7 @@ static int dix_cas_rops_send(struct m0_dix_req *req)
 		default:
 			M0_IMPOSSIBLE("Unknown req type %u", req->dr_type);
 		}
+		m0_cas_id_fini(&cctg_id);
```

Comment:
This frees used data: m0_cas_get()->m0_cas__get()->cas_req_prep()->cas_records_op_prepare() copies cas_id:

op->cg_id = *index;

This means that extent op->cg_id.ci.layout.dl_desc.ld_imask.im_range is shared between op->cg_id and cctg_id. m0_cas_id_fini() frees the extent, so garbage is written in fop and sent to the server.

Please revert.

Reply:
Opened https://github.com/Seagate/cortx-motr/issues/1601 for this. Thanks!